libs/orders/src/lib/components/order-list/order-edit-dialog.tsx

```diff
@@ -88,15 +88,23 @@ export const OrderEditDialog = ({
         <form onSubmit={handleSubmit(onSubmit)} data-testid="edit-order">
           <FormGroup label={t('Entry price')} labelFor="entryPrice">
             <Input
```

Comment:
question (non-blocking): not sure if we need to / want to add the market decimalPlaces so that the input `step` can reflect the minimum unit of change

Reply:
this is how it looks like after i pulled latest changes on master

Reply:
this looks more like a side-effect of changing the input type from "text" to "number". number types need a "step" attribute in order to accept decimal values

Reply:
updated